File: ITSC 3155/itsc-3155/m1/syntax/logic/conditionals.java
// a simple review of if statements in Java
package m1.syntax.logic;

public class conditionals {
    public static void main(String[] args) {
        int x = 5;
        int y = 10;
        int z = 15;

        if (x > y) {
            System.out.println("x is greater than y");
        } else if (x < y) {
            System.out.println("x is less than y");
        } else {
            System.out.println("x is equal to y");
        }

        if (x > y && x > z) {
            System.out.println("x is the greatest");
        } else if (y > x && y > z) {
            System.out.println("y is the greatest");
        } else if (z > x && z > y) {
            System.out.println("z is the greatest");
        } else {
            System.out.println("x, y, and z are equal");
        }
    }

}

File: ITSC 3155/itsc-3155/m1/syntax/logic/conditionals.py
# a simple review of if statements in Python

def main():
    x = 5
    y = 10
    z = 15

    if x > y:
        print("x is greater than y")
    elif y > x:
        print("x is less than y")
    else:
        print("x is equal to y")

    if x > y and x > z:
        print("x is the greatest")
    elif y > x and y > z:
        print("y is the greatest")
    elif z > x and z > y:
        print("z is the greatest")
    else:
        print("x, y, and z are equal")


if __name__ == "__main__":
    main()
